internal/operands/metrics/resources.go

```diff
@@ -45,7 +47,9 @@ func newPrometheusRule(namespace string) *promv1.PrometheusRule {
 								"runbook_url": "https://kubevirt.io/monitoring/runbooks/SSPOperatorDown",
 							},
 							Labels: map[string]string{
-								"severity": "critical",
+								"severity":                      "critical",
+								"kubernetes_operator_part_of":   SSPOperatorPartOf,
+								"kubernetes_operator_component": SSPOperatorComponent,
```

Comment:
Can you also extract the keys to constants (`severity`, `kubernetes_operator_part_of`, `kubernetes_operator_component`) ? So we can be sure there is not a typo in the strings used in multiple places.

Reply:
Sure, I'll do it in a separate PR.

Reply:
Why in a separate PR? They are added in this PR, so you can add them as constants instead of as literals?

Reply:
Hi @erkanerol, do you think extracting the keys into constants makes sense in the case?